src/main/java/core/basesyntax/SaveData.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class SaveData {
+    public void writeData(String fileName, String saveData) {
+        try (var fileWriter = new BufferedWriter(new FileWriter(fileName))) {
```

Comment:
```suggestion
        try (FileWriter fileWriter = new BufferedWriter(new FileWriter(fileName))) {
```

Reply:
I can't rename class WorkWithFile
I got it with task and this class name is using in test

Reply:
Omg -.-
No problem, let's leave it alone)